developer-workflow/development-cycle.rst

```diff
@@ -67,6 +67,10 @@ for release 3.n.
 Maintenance branches
 --------------------
 
+.. note:: Any documentation and test improvements should be backported to enhance currently
+    **stable** versions. This includes any branch currently in a release cycle of
+    **maint** (**bugfix**) and above.
```

Comment:
Backporting doc is to make doc changes more visible, since the majority of users visit docs.python.org/3/ or a specific version, not /dev/ doc. According to stats.

Bakcporting tests is to reduce conflicts when bugfixes are backported later.

Reply:
Would it be possible to write a regular paragraph, rather than a note?

Reply:
I'd write this as regular prose, but also note that line 83 already contains text about backporting, so we should really make the change in that para, or nearby.

Reply:
Maybe it seeming to be still around was part of my force push? Sorry 😦